### ISSUE TITLE
[Bug Report] search works at the second attempt sometimes

### ISSUE THREAD

**Issue by eyehol3** [jagrosh/MusicBot#443]
**Describe the bug**

i type `play [request]` first time
\- No results found
i repeat same request:
\- Playing [smth]

**To Reproduce**
1. Run with `java -Dnogui=true -jar JMusicBot-0.2.9-Mac.jar`
2. Do a command `play [request]`

**Expected behavior**
results are found at the first attempt

**Screenshots**
![image](https://user-images.githubusercontent.com/49121161/84570189-70f39c80-ad94-11ea-8870-95ec055f2bd5.png)

**Comment by Phillackinger** [comment]
![image](https://user-images.githubusercontent.com/40075339/84604331-813f7080-ae95-11ea-953e-af618e66a7b8.png)

I have the same Issue. Only this time it doesn't find anything with $search but on the second attempt the bots finds it with $play

**Comment by alfonsojon** [comment]
**Describe the bug**
Occasionally, the bot fails to search for a song almost as though no results came up.

**To Reproduce**
Steps to reproduce the behavior:
1. Search for the same song many times for the sake of testing
2. Notice it occasionally fails

**Expected behavior**
I would assume it should successfully search every time.

**Screenshots**
![image](https://user-images.githubusercontent.com/1163298/84683682-335e5300-aefd-11ea-8df5-7cd24563daf8.png)


**Debug Info**
```
System Properties:
  java.version = 1.8.0_252
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 1.8
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 1.8.0_252-8u252-b09-1~18.04-b09
  java.specification.version = 1.8
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = Snapshot
  Owner = 150301926821462016
  Prefix = @mention
  AltPrefix = m!
  MaxSeconds = 900
  NPImages = false
  SongInStatus = true
  StayInChannel = false
  UseEval = false
  UpdateAlerts = false

Dependency Information:
  JDA Version = 3.8.3_464
  JDA-Utilities Version = 2.1.5
  Lavaplayer Version = 1.3.49

Runtime Information:
  Total Memory = 135
  Used Memory = 92

Discord Information:
  ID = 666773938860458009
  Guilds = 1
  Users = 6332
```

**Additional context**
I am running a JAR compiled from source to resolve an issue with fetching most songs; however, I have made no alterations to the source code and this issue has impacted the past two versions (or more) of the music bot.

**Comment by Sanduhr32** [comment]
Self compiled copies aren't supported.

However why did you expect it to resolve the issue if no changes to the build file, libraries nor source code was made?

Java should not be affected by compiling the code on different operating systems nor cpu vendors / architectures.

JMusicbot uses Lavaplayer which comes with pre-built binaries for different operating systems and cpu architectures.
If you believe the binary has an issue or isn't compatible then you might have to use a self compiled version of Lavaplayer resulting in a again unsupported self compiled JMusicbot.

**Comment by alfonsojon** [comment]
The jar I have is just 0.28 with an updated pom.xml to use lavaplayer 1.3.49. I have since updated to 0.29 as I saw that was out. @eyehol3 has reported the same issue on version 0.29 so it seems to still be an issue.

**Comment by Sanduhr32** [comment]
If you compare [0.2.8 to 0.2.9 (this is a github link)](https://github.com/jagrosh/MusicBot/compare/0.2.8...0.2.9) then you see that there isn't a big change in JMusicbot, and that it might have a different source.

**Comment by alfonsojon** [comment]
![image](https://user-images.githubusercontent.com/1163298/85226958-cdac1400-b39f-11ea-91c8-ddda9614824d.png)
screenshot taken on 0.2.9

**Comment by nexusdiscord** [comment]
![Untitled](https://user-images.githubusercontent.com/63144643/85673717-5ef7ef00-b6ee-11ea-8f09-28546fb579ae.png)

I have same issue

**Comment by hondasila** [comment]
![image](https://user-images.githubusercontent.com/56035181/85807429-2ee23780-b752-11ea-82cb-7a3720f990ae.png)
same

**Comment by iRealKsA** [comment]
![image](https://user-images.githubusercontent.com/48433959/85818606-b42d1280-b779-11ea-8dfa-258e5368c726.png)
???????????? Please Fix it

**Comment by spiderboyjk8** [comment]
Same issue please fix!!!

**Comment by onlyyouX2** [comment]
Mr. jagrosh 

The same problem , I think you should add YouTube search it would be better
Thank you .

**Comment by Sanduhr32** [comment]
YouTube changed the format of how it sends the data.
This issue should be fixed in the next version of JMusicbot.

**Comment by Djinnistorm** [comment]
yea it's fixed in 1.3.50 version of lavaplayer, so rebuilding with that yourself or waiting are your two options.

**Comment by onlyyouX2** [comment]
Waiting for update Version 0.3.0 , I used another  bot until the new version come out

**Comment by nexusdiscord** [comment]
> yea it's fixed in 1.3.50 version of lavaplayer, so rebuilding with that yourself or waiting are your two options.

still getting problem

**Comment by hondasila** [comment]
> > yea it's fixed in 1.3.50 version of lavaplayer, so rebuilding with that yourself or waiting are your two options.
> 
> still getting problem

do you rebuild it by yourself ?

**Comment by nexusdiscord** [comment]
> > > yea it's fixed in 1.3.50 version of lavaplayer, so rebuilding with that yourself or waiting are your two options.
> > 
> > 
> > still getting problem
> 
> do you rebuild it by yourself ?

yes

**Comment by hondasila** [comment]
> yes
i only cant convert .java (lavaplayerfile) to a .class to change it

**Comment by jagrosh** [comment]
Building the project yourself is not supported; please be patient for the next release, which will come out once lavaplayer's fix for this issue is stable.

**Comment by jagrosh** [comment]
This should be resolved as of [version 0.2.10](https://github.com/jagrosh/MusicBot/releases/latest)
